app.go

```diff
@@ -238,6 +240,7 @@ type CheckPushResponse struct {
 	DeviceErrors             map[string]DeviceError `json:"device_errors"`
 	GeneralErrors            []string               `json:"general_errors"`
 	SkipDevices              *bool                  `json:"skip_devices"`
+	ProviderName             string                 `json:"provider_name"`
```

Comment:
this is in wrong place, will be on `DeviceError` since each device can have a different provider

Reply:
you're right! fixed